internal/stats/metrics_recorder_list.go

```diff
@@ -49,50 +49,46 @@ func NewMetricsRecorderList(shs []stats.Handler) *MetricsRecorderList {
 	}
 }
 
-func (l *MetricsRecorderList) RecordInt64Count(handle *estats.Int64CountHandle, incr int64, labels ...string) {
-	if got, want := len(handle.Labels)+len(handle.OptionalLabels), len(labels); got != want {
-		logger.Infof("length of labels passed to RecordInt64Count incorrect got: %v, want: %v", got, want)
+func verifyLabels(labels []string, optionalLabels []string, labelsRecv ...string) {
```

Comment:
Could this pass a `*MetricDescriptor` instead of the labels/optionallabels fields?

Reply:
Done.